randovania/data/json_data/prime1.txt

```diff
@@ -2403,582 +2411,1261 @@ Ruins Entrance
 > Dock to Burn Dome Access; Heals? False
-  * Normal Door to Burn Dome Access/Dock to Energy Core
+  * Morph Ball Door to Burn Dome Access/Dock to Energy Core
+  > Dock to Energy Core Access
+      Trivial
+  > Event - Unlock West Furnace Access door
+      Morph Ball Bomb and Morph Ball
 
 > Dock to West Furnace Access; Heals? False
   * Normal Door to West Furnace Access/Dock to Energy Core
+  > Dock to Energy Core Access
+      Trivial
 
 > Dock to Energy Core Access; Heals? False
   * Normal Door to Energy Core Access/Dock to Energy Core
+  > Dock to Burn Dome Access
+      Trivial
+  > Dock to West Furnace Access
+      After Unlock West Furnace Access door
+
+> Event - Unlock West Furnace Access door; Heals? False
+  * Event Unlock West Furnace Access door
+  > Dock to Energy Core Access
+      Trivial
 
 ----------------
 Dynamo
 Asset id: 81183365
 > Dock to Dynamo Access; Heals? False
-  * Normal Door to Dynamo Access/Dock to Dynamo
+  * Missile Blast Shield to Dynamo Access/Dock to Dynamo
+  > Pickup (Missile Expansion)
+      Missile
+  > Pickup (Missile Expansion 2)
+      Morph Ball and Spider Ball
+
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 25; Major Location? False
+  > Dock to Dynamo Access
+      Trivial
+
+> Pickup (Missile Expansion 2); Heals? False
+  * Pickup 26; Major Location? False
+  > Dock to Dynamo Access
+      Trivial
 
 ----------------
 Burn Dome Access
 Asset id: 4018058640
 > Dock to Burn Dome; Heals? False
   * Normal Door to Burn Dome/Dock to Burn Dome Access
+  > Dock to Energy Core
+      Morph Ball Bomb and Morph Ball
 
 > Dock to Energy Core; Heals? False
   * Normal Door to Energy Core/Dock to Burn Dome Access
+  > Dock to Burn Dome
+      Morph Ball
 
 ----------------
 West Furnace Access
 Asset id: 3262208600
 > Dock to Energy Core; Heals? False
   * Normal Door to Energy Core/Dock to West Furnace Access
+  > Dock to Furnace
+      Trivial
 
 > Dock to Furnace; Heals? False
   * Normal Door to Furnace/Dock to West Furnace Access
+  > Dock to Energy Core
+      Trivial
 
 ----------------
 Burn Dome
 Asset id: 1095301040
 > Dock to Burn Dome Access; Heals? False
   * Normal Door to Burn Dome Access/Dock to Burn Dome
+  > Pickup (Morph Ball Bombs)
+      Trivial
+  > Pickup (Missile Expansion)
+      Morph Ball Bomb and Morph Ball
+
+> Pickup (Morph Ball Bombs); Heals? False
+  * Pickup 28; Major Location? True
+  > Dock to Burn Dome Access
+      Trivial
+
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 27; Major Location? False
+  > Dock to Burn Dome Access
+      Morph Ball Bomb and Morph Ball
 
 ----------------
 Furnace
 Asset id: 774997107
 > Dock to East Furnace Access; Heals? False
-  * Normal Door to East Furnace Access/Dock to Furnace
+  * Ice Door to East Furnace Access/Dock to Furnace
+  > Dock to West Furnace Access
+      Morph Ball Bomb and Morph Ball
+  > Pickup (Energy Tank)
+      Morph Ball Bomb and Morph Ball
+  > Half pipe
+      Power Bomb and Morph Ball
```

Comment:
Remove connections to etank and half pipe. Change door connection to the helper node with trivial requirement.

Reply:
Done